package.json

```diff
@@ -88,29 +88,29 @@
     "enzyme-adapter-react-16": "1.15.2",
     "eslint": "6.8.0",
     "eslint-config-prettier": "6.9.0",
-    "eslint-plugin-jest": "23.3.0",
+    "eslint-plugin-jest": "23.6.0",
     "eslint-plugin-prettier": "3.1.2",
-    "eslint-plugin-react": "7.17.0",
+    "eslint-plugin-react": "7.18.0",
     "eslint-plugin-react-hooks": "2.3.0",
     "fake-indexeddb": "3.0.0",
     "hard-source-webpack-plugin": "0.13.1",
     "html-webpack-plugin": "3.2.0",
-    "husky": "4.0.1",
+    "husky": "4.0.10",
     "jest": "24.9.0",
     "jest-localstorage-mock": "2.4.0",
     "mini-css-extract-plugin": "0.9.0",
-    "node-sass": "4.13.0",
+    "node-sass": "4.13.1",
     "postcss-loader": "3.0.0",
     "prettier": "1.19.1",
     "pretty-quick": "2.0.1",
     "react-onclickoutside": "6.9.0",
     "react-test-renderer": "16.12.0",
     "rimraf": "3.0.0",
-    "sass-loader": "8.0.0",
+    "sass-loader": "8.0.2",
     "spectron": "6.0.0",
-    "style-loader": "1.1.2",
-    "typescript": "3.7.4",
-    "wait-on": "3.3.0",
+    "style-loader": "1.1.3",
+    "typescript": "3.7.5",
+    "wait-on": "4.0.0",
```

Comment:
do we even need this? this is a major update so I checked to see what we'd need to change and it's not referenced anyway. it came in #859 with `misty` and I'm not sure if it was meant to be a transitive/peer dependency. I haven't tried building without it

Reply:
https://github.com/Automattic/simplenote-electron/blob/develop/misty.yml#L6

Reply:
It is used.

Reply:
I don't think the API changed but they rewrote the code thus the full point bump.